### PR TITLE
Fix UI issues on frontend

### PR DESCRIPTION
**Pull request by totop716**
- Hide scrollbar on tradingview chart on swap pro mode screenshot
- Fix token logo on analytics pages.
- Update font on search input on farms page. screenshot1
- Fix exchange price section on swap component.  screenshot
- Fix expert confirm input on settings modal. screenshot
- Call exit function when user withdraw whole amount from the farm. screenshot
- Update liquidity pools on swap page.  screenshot1
- Fix token pairs on analytics token details.  screenshot2

### CODE REVIEW

File: src/apollo/queries.ts [hunk 1]
```diff
@@ -426,10 +426,10 @@ export const TOKEN_DATA1: any = (
 export const TOKEN_DATA2: any = (tokenAddress: string) => {
   const queryString = `
     query tokens {
-      pairs0: pairs(where: {token0: "${tokenAddress}"}, first: 50, orderBy: reserveUSD, orderDirection: desc){
+      pairs0: pairs(where: {token0: "${tokenAddress}"}, first: 50, orderBy: trackedReserveETH, orderDirection: desc){
         id
       }
-      pairs1: pairs(where: {token1: "${tokenAddress}"}, first: 50, orderBy: reserveUSD, orderDirection: desc){
+      pairs1: pairs(where: {token1: "${tokenAddress}"}, first: 50, orderBy: trackedReserveETH, orderDirection: desc){
```
Comment (sameepsi) [inline src/apollo/queries.ts:432]: Why have you changed this query?

Comment (totop716) [reply src/apollo/queries.ts:432]: We are calculating liquidity using `trackedReservedETH` and made this change in order to show correct pairs.

Comment (FriskyHamTitz) [inline src/apollo/queries.ts:432]: We sure these queries are used else where? Where it's expecting the the reserveUsd

File: src/components/AnalyticsSearch/index.tsx [hunk 2]
```diff
@@ -283,15 +286,29 @@ const AnalyticsSearch: React.FC = () => {
...
+            const currency1 = getTokenFromAddress(
+              val.token1.id,
               MATIC_CHAIN,
-              getAddress(val.token1.id),
-              val.token1.decimals,
+              tokenMap,
+              [
+                new Token(
+                  MATIC_CHAIN,
+                  getAddress(val.token1.id),
```
Comment (FriskyHamTitz) [inline src/components/AnalyticsSearch/index.tsx:308]: Why do we need to call get address if you just grabbed the token from the address

File: src/components/PairsTable/PairsTable.tsx [hunk 3]
```diff
@@ -69,18 +70,33 @@ const PairTable: React.FC<PairsTableProps> = ({
...
+      pair.token0.id,
       ChainId.MATIC,
-      getAddress(pair.token0.id),
-      Number(pair.token0.decimals),
-      pair.token0.symbol,
+      tokenMap,
+      [
+        new Token(
+          ChainId.MATIC,
+          getAddress(pair.token0.id),
```
Comment (FriskyHamTitz) [inline src/components/PairsTable/PairsTable.tsx:82]: Isn't the I'd already the address?

File: src/pages/AnalyticsPairDetails/AnalyticsPairDetails.tsx [hunk 4]
```diff
@@ -61,18 +64,22 @@ const AnalyticsPairDetails: React.FC = () => {
...
   const currency0 = pairData
-    ? new Token(
-        ChainId.MATIC,
-        getAddress(pairData.token0.id),
-        pairData.token0.decimals,
-      )
+    ? getTokenFromAddress(pairData.token0.id, ChainId.MATIC, tokenMap, [
+        new Token(
+          ChainId.MATIC,
+          getAddress(pairData.token0.id),
```
Comment (FriskyHamTitz) [inline src/pages/AnalyticsPairDetails/AnalyticsPairDetails.tsx:70]: Fix in all places

**Review by FriskyHamTitz — CHANGES_REQUESTED** [review]

File: src/utils/index.ts [hunk 5]
```diff
@@ -1743,7 +1745,7 @@ export function getTokenFromAddress(
     return token;
   }
 
-  return wrappedTokenInfo;
+  return Object.values(tokenMap[chainId])[tokenIndex];
```
Comment (FriskyHamTitz) [inline src/utils/index.ts:1748]: We're sure this isn't used as a dependency in any hooks cuz it will cause an infinite loops if so

**Review by FriskyHamTitz — APPROVED** [review]

File: src/utils/index.ts (hunk 5 above)
Comment (totop716) [reply src/utils/index.ts:1748]: This is not used in hook so it will not cause infinite loop.

**Pull request merged**